pkg/app/app.go

```diff
@@ -530,6 +530,23 @@ func (a *App) Test(c TestConfigProvider) error {
 	}, false, SetFilter(true))
 }
 
+func (a *App) PrintDAGState(c DAGConfigProvider) error {
+	var err error
+	return a.ForEachState(func(run *Run) (ok bool, errs []error) {
+		err = run.withPreparedCharts("show-dag", state.ChartPrepareOptions{
+			SkipRepos:   c.SkipDeps(),
+			SkipDeps:    c.SkipDeps(),
+			Concurrency: c.Concurrency(),
+		}, func() {
```

Comment:
@vlpav030 oh. sorry. I think same as `build` subcommand is ok. so you can revert your code. I will merge code after revertd.